### PR TITLE
#7 | Fixes Deathspitter cost for Tyranid Warrior

### PR DESCRIPTION
**Pull request by DevJonny**
Adds a cost override for Tyranid Warrior so it's 2pts.

### CODE REVIEW

File: src/factions/Tyranids/models.json [hunk 1]
```diff
@@ -870,6 +870,10 @@
           {
             "WeaponId": "DEV",
             "Cost": 0
+          },
+          {
+            "WeaponId": "CRM",
+            "Cost": 2
```
Comment (jakemco) [inline src/factions/Tyranids/models.json:876]: This should probably be modeled as the base cost and the other ones as the override (like it shows in the annual), but this is "good enough" for now to get the points in the right spot.

**Review by jakemco — APPROVED** [review]
If you put "fixes #7" instead of just referencing the number, it'll autoclose the issue. You can also put it in the summary rather than the commit message.

**Comment by DevJonny** [comment]
> If you put "fixes #7" instead of just referencing the number, it'll autoclose the issue. You can also put it in the summary rather than the commit message.

Yeah I couldn't remember how to get GitHub to pick up on it. Jira just needs the ticket somewhere in the title. I'll add do the Fixes #7 when I squash.

File: src/factions/Tyranids/models.json (hunk 1 above)
Comment (DevJonny) [reply src/factions/Tyranids/models.json:876]: Yeah that's the the thing, it was the base cost and the Warrior wasn't picking it up! Hence the override.

Comment (DevJonny) [reply src/factions/Tyranids/models.json:876]: https://github.com/KTManager/KTRules/blob/2c719e1e024438379b1bf524484d16ef342f6f6a/src/weapons.jsonnet#L2589

**Pull request merged**